src/screens/Series/Series.tsx

```diff
@@ -33,19 +32,17 @@ import FavoriteButton from '#src/containers/FavoriteButton/FavoriteButton';
 import Button from '#src/components/Button/Button';
 import PlayTrailer from '#src/icons/PlayTrailer';
 
-type SeriesRouteParams = {
-  id: string;
-};
-
-const Series = ({ match, location }: RouteComponentProps<SeriesRouteParams>): JSX.Element => {
+const Series = (): JSX.Element => {
   const breakpoint = useBreakpoint();
   const { t } = useTranslation('video');
   const [playTrailer, setPlayTrailer] = useState<boolean>(false);
 
   // Routing
-  const history = useHistory();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const params = useParams();
   const searchParams = useMemo(() => new URLSearchParams(location.search), [location.search]);
-  const id = match?.params.id;
+  const id = params.id as string;
```

Comment:
Prevent type casting, because `params.id` can be undefined.

```suggestion
  const id = params.id || '';
```